app/helpers/ops_helper/textual_summary.rb

```diff
@@ -163,7 +163,13 @@ def subprojects_from_record
 
   def textual_tenant_quota_allocations
     h = {:title     => _("Tenant Quota"),
-         :headers   => [_("Name"), _("Total Quota"), _("In Use"), _("Allocated"), _("Available")],
+         :headers   => [
+           {:key => :name, :label => _("Name")},
+           {:key => :total_quota,  :label => _("Total Quota")},
```

Comment:
Super minor alignment issue - doesn't need to hold up the merge.
```suggestion
           {:key => :name,         :label => _("Name")},
           {:key => :total_quota,  :label => _("Total Quota")},
```